NESTED-VALIDATION.md

```diff
@@ -88,15 +92,15 @@ The GnosisSafe `approvedHashes` mapping is updated to indicate approval of this
     ```
     The output of this command must match the key of the state change.
 
-### Liveness Guard
+### Liveness Guard (Security Council only)
 
-When the Security Council executes a transaction, the liveness timestamp are updated for each owner that signed the tasks.
-This is updating at the moment of the transaction is submitted (`block.timestamp`) into the [`lastLive`](https://github.com/ethereum-optimism/optimism/blob/e84868c27776fd04dc77e95176d55c8f6b1cc9a3/packages/contracts-bedrock/src/safe/LivenessGuard.sol#L41) mapping located at the slot `0`.
+When the Security Council executes a transaction, the liveness timestamps are updated for each owner that signed the task.
+This is updating at the moment when the transaction is submitted (`block.timestamp`) into the [`lastLive`](https://github.com/ethereum-optimism/optimism/blob/e84868c27776fd04dc77e95176d55c8f6b1cc9a3/packages/contracts-bedrock/src/safe/LivenessGuard.sol#L41) mapping located at the slot `0`.
 
 ### Nonce increments
 
 The only other state changes related to the nested execution are _three_ nonce increments:
 
 - One on the `ProxyAdminOwner` 2/2. If this is not decoded, it corresponds to key `0x05` on a `GnosisSafeProxy`.
```

Comment:
```suggestion
- One increment on the slot of the  `ProxyAdminOwner` 2/2 also known as (L1PAO), located at the storage key:  `0x0000000000000000000000000000000000000000000000000000000000000005` of the `Safe`.
```

Reply:
I think it's a matter of taste. I like short sentences of rather trivial facts over long sentences that require more brain-parsing 😅 I also deliberately didn't mention the L1PAO or BL1PAO here again, because the `ProxyAdminOwner` is already defined at the top. This just reduces the chance of us forgetting to update this doc here when we add additional Safes in the future. I'll still change it to be slightly more descriptive.

Reply:
Ok sounds good to me thanks for the explanation.